digdag-core/src/main/java/io/digdag/core/database/DatabaseTaskQueueServer.java

```diff
@@ -441,6 +446,7 @@ void expireLocks()
         catch (Throwable t) {
             logger.error("An uncaught exception is ignored. This lock expiration thread will be restarted.", t);
             errorReporter.reportUncaughtError(t);
+            metrics.increment(Category.DEFAULT, "uncaughtErrors");
```

Comment:
Thanks for your pointing out.
In JMX, DEFAULT is mapping to the domain "io.digdag" and used for "misc" metrics.
I set DEFAULT because I thought it was different from DB access.
But it is still under database package name space.
I will change to "DB".

Reply:
Fixed it.

Reply:
thanks 👍